src/inputUrls.txt

```diff
@@ -0,0 +1,24 @@
+https://matchhistory.na.leagueoflegends.com/en/#match-details/NA1/3609773279/48333464?tab=overview
```

Comment:
why do we want to merge this?

Reply:
we can keep this running list of urls so anyone can recompile all the matches, which we need to do if we add scraping of more specific match data

Reply:
sounds good! can we move it to a different folder? (data/ or similar)